samples/nrf9160/custom_trace_medium/prj.conf

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# General config
+CONFIG_NEWLIB_LIBC=y
+
+# Modem library
+CONFIG_NRF_MODEM_LIB=y
+CONFIG_NRF_MODEM_LIB_SYS_INIT=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+
+# Modem trace
+CONFIG_NRF_MODEM_LIB_TRACE_ENABLED=y
+CONFIG_NRF_MODEM_LIB_TRACE_THREAD_PROCESSING=y
```

Comment:
@lemrey depends on what you are referring to. If the question is why are these kconfigs enabled, then it's because the tracing, with thread processing, is required to be able to execute the custom trace function calls, which give an output that we want to see in the sample.

Reply:
That's right, I got confused with the other PR - thanks.